### ISSUE TITLE
Follow the guidelines for creating an IANA registry

### ISSUE THREAD

**Issue by jyasskin** [w3c-ccg/multibase#6]
https://www.iana.org/help/protocol-registration and https://www.rfc-editor.org/rfc/rfc8126.html say how to create IANA registries, and the current draft is missing some important parts, like the registration policy.

**Comment by bumblefudge** [comment]
Agreed; the way I'm reading [section 2.3](https://www.rfc-editor.org/rfc/rfc8126.html#section-2.3) is that if IANA isn't hosting the party, it should be recognizably and explicitly equivalent. I will try to PR in some IANA-analogous change policies this week
